tests/pages/checkout_page.py

```
@@ -26,95 +25,158 @@ def click_checkout(self, flow="general"):
+    # ---------------------------------------------------
+    # Actions
+    # ---------------------------------------------------
+
+    def click_continue_button(self, timeout=20):
+        wait = self.get_wait(timeout)
+
+        continue_btn = wait.until(
+            EC.presence_of_element_located(self.locators.CONTINUE_BUTTON)
+        )
+
+        # ✅ enabled olmasını bekle (CRITICAL)
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF003)